Readme.md

```diff
@@ -50,7 +47,7 @@ Let's assume you have an object with `title` and `slug` fields. Before applying
 }
 ```
 
-Now, let's consider you are adding two languages: `pl` and `en`, and your default language is `en`. You are choosing the `title` field to translate. After applying translations, the object will be:
+Now, let's consider you are adding two languages: `Poish` and `English`, and your default language is `English`. You are choosing the `title` field to translate. After applying translations, the object will be:
```

Comment:
```suggestion
Now, let's consider that you are adding two languages: `Polish` and `English`, and your default language is `English`. After applying translations, the object will be:
```
Typa, and we no longer choose the fields. Also, the `slug` should be in the `__translations` object.